src/UI/webWallet/WebWalletLoginButton/WebWalletLoginButton.tsx

```diff
@@ -25,12 +27,14 @@ export const WebWalletLoginButton: (
   nativeAuth,
   'data-testid': dataTestId,
   loginButtonText = 'MultiversX Web Wallet',
-  disabled
+  disabled,
+  customWalletAddress
```

Comment:
Maybe the property description would help here like in `AccountInfoSliceNetworkType`